### ISSUE TITLE
FR: Editing Inserted page without need to open it

### ISSUE THREAD

**Issue by Deus333** [zsviczian/obsidian-excalidraw-plugin#1475]
Being able to edit an inserted excalidraw drawing directly from another excalidraw drawing that it inserted in without need to open it like a frame

**Comment by zsviczian** [comment]
yes - would be great but I ran into technical difficulties with this that I wasn't able to resolved. So opening in another tab is your best option currently.
